src/utils/tokens.ts

```diff
@@ -462,6 +539,28 @@ const tokens = {
       });
     }
 
+    if (data !== null) {
+      for (const dataString of data) {
+        const outputData = {
+          type: 'data',
+          data: dataString,
+          value: 1,
+          token: HATHOR_TOKEN_CONFIG.uid,
+          authorities: 0,
+        } as IDataOutput;
+
+        // We currently have an external service that identifies NFT tokens with the first output as the data output
+        // that's why we are keeping like this
+        // However, this will change after a new project is completed to better identify an NFT token
+        // the method that validates the NFT is in src/models/CreateTokenTransaction.validateNft
```

Comment:
We can keep the `unshift` parameter for consistency but this comment is not needed in this case. It's needed in the other because it's used for creating an NFT but here we don't create a token, it's just melt.

Reply:
Done